src/main/scala/EnsimePlugin.scala

```diff
@@ -30,6 +30,9 @@ object Imports {
     val additionalCompilerArgs = TaskKey[Seq[String]](
       "Additional arguments for the presentation compiler."
     )
+    val javaFlags = TaskKey[Seq[String]](
```

Comment:
Oh, I thought in the original ticket that's what you wanted, no?
https://github.com/ensime/ensime-sbt/issues/159